### ISSUE TITLE
Togglable Gyro Controls

### ISSUE THREAD

**Issue by JellyFlame** [mfosse/JoyCon-Driver#69]
I wanted to emulate a Wiimote in dolphin so for tilt I used the Sliders (Slider 1+ and Slider 1-) , but I noticed once you tilted the Joycon it would return to its previous position. It would be helpful to be able to toggle the tilt or any gyro controls so when I have the Joycon tilted the dot stays tilted. Sorry if this explanation makes no sense and thank you so much for making this driver.
![gyro controls](https://user-images.githubusercontent.com/37758439/37870614-4b5e455e-2f8f-11e8-935f-3eec0911e837.png)

**Comment by mfosse** [comment]
Working on it :P

**Comment by mfosse** [comment]
I've added a new way to toggle gyro controls, but I'll leave the issue open for now since the feature still needs some work, but let me know how well it works for you @JellyFlame

**Comment by JellyFlame** [comment]
When I checked the "Quick Toggle Gyro Controls" box nothing changed. It still resets to 0 or the middle after I move the Joycon.

**Comment by mfosse** [comment]
Oh, I see now, the title for the issue and what I think you're trying to achieve are different things.

while I don't have a way to make the gyro controls work the way you want right now (although it could be done) it's easier to set dolphin to use the mouse as the wii pointer, and then enable gyro controls, which should do exactly what you want

**Comment by JellyFlame** [comment]
Alright well sorry for not explaining so well and thanks for the advise.

**Comment by rccastruita** [comment]
Hi. Everything seems to work just fine with the gyro controls. At least in my case, the B button is the one that toggles the gyro controls. Is there a way to change the button for toggling gyro?

**Comment by mfosse** [comment]
@rccastruita Yes, the gyroscope combo code in the config file corresponds to the button(s) that toggle the gyro controls

To figure out what number to put there, run the program and look for the number next to gyro combo code when you press the button(s) you want

**Comment by revitup360** [comment]
Is it possible to have gyroscope controls activate for as long as a button is held? For instance, holding ZL enables gyro and releasing it disables it.

**Comment by mfosse** [comment]
@revitup360 yeah sure I'll make it happen in the next update
